e2e/e2e_test.go

```diff
@@ -62,45 +60,47 @@ func TestDataDealValidateData(t *testing.T) {
 	require.Equal(t, data, string(decryptedData))
 }
 
-func TestDataPoolValidateData(t *testing.T) {
-	buyerMnemonic := os.Getenv("E2E_DATA_BUYER_MNEMONIC")
-	require.NotEmpty(t, buyerMnemonic)
-	dataValMnemonic := os.Getenv("E2E_DATAVAL_MNEMONIC")
-	require.NotEmpty(t, dataValMnemonic)
-	datavalHTTPAddr := os.Getenv("E2E_DATAVAL_HTTP_ADDR")
-	require.NotEmpty(t, datavalHTTPAddr)
-
-	poolID := 1
-	round := 1
-	requesterAddr := "panacea1c7yh0ql0rhvyqm4vuwgaqu0jypafnwqdc6x60e"
-	data := `{
-		"name": "This is a name",
-		"description": "This is a description",
-		"body": [{ "type": "markdown", "attributes": { "value": "val1" } }]
-	}`
-
-	req, err := http.NewRequest(
-		http.MethodPost,
-		fmt.Sprintf("http://%s/v0/data-pool/pools/%d/rounds/%d/data?requester_address=%s", datavalHTTPAddr, poolID, round, requesterAddr),
-		strings.NewReader(data),
-	)
-	require.NoError(t, err)
-	req.Header.Set("Content-Type", "application/json")
-
-	resp, err := http.DefaultClient.Do(req)
-	require.NoError(t, err)
-	defer resp.Body.Close()
-	require.Equal(t, http.StatusCreated, resp.StatusCode)
-
-	var cert datapooltypes.DataValidationCertificate
-	unmarshaler := &jsonpb.Unmarshaler{}
-	err = unmarshaler.Unmarshal(resp.Body, &cert)
-	require.NoError(t, err)
-
-	// TODO Check if MED is sent to the seller normally after the sale data.
-
-	// TODO Confirm that the buyer who purchased NFT Token can import data normally.
-}
+// TODO: When the data pool module is done, the e2e unit test will be added.
+//func TestDataPoolValidateData(t *testing.T) {
+//	buyerMnemonic := os.Getenv("E2E_DATA_BUYER_MNEMONIC")
+//	require.NotEmpty(t, buyerMnemonic)
```

Comment:
The contract should be instantiated through governance before create pool in panacea-core so that I marked this function `TODO`.